Source/Xamarin/Prism.Forms/Services/PageDialogService/ActionSheetButton.cs

```diff
@@ -1,74 +1,201 @@
-﻿using System.Windows.Input;
+﻿using System;
+using System.Windows.Input;
 
 namespace Prism.Services
 {
     /// <summary>
     /// Represents a button displayed in <see cref="Prism.Services.IPageDialogService.DisplayActionSheetAsync(string, IActionSheetButton[])"/>
     /// </summary>
-    public class ActionSheetButton : IActionSheetButton
+    public class ActionSheetButton : ActionSheetButtonBase
     {
+        protected internal ActionSheetButton() { }
+
         /// <summary>
-        /// The button will be used as destroy
+        /// Action to perform when the button is pressed
         /// </summary>
-        public virtual bool IsDestroy { get; protected set; }
+        /// <value>The action.</value>
+        public Action Action { get; protected set; }
+
+        /// <inheritDoc />
```

Comment:
Currently C# doesn't support this tag, Sandcastle does.
https://visualstudio.uservoice.com/forums/121579-visual-studio/suggestions/3745102-add-intellisense-support-for-the-inheritdoc-ta